### PR TITLE
Archived projects layout [SCI-5266]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-5266](https://biosistemika.atlassian.net/browse/SCI-5266)

### What was done
Added archived mode for projects, moved navigation

### CODE REVIEW

File: app/assets/javascripts/projects/index.js [hunk 1]
```diff
@@ -563,28 +561,32 @@
...
-      selectedProjects = [];
-      projectsViewFilter = $(this).data('filter');
-      loadCardsView();
-    });
-  }
-
   function initProjectsViewModeSwitch() {
-    $('input[type=radio][name=projects-view-mode-selector]').change(function() {
-      if (this.value === 'cards') {
+    let projectsPageSelector = '.projects-index ';
```
Comment (aignatov-bio) [inline app/assets/javascripts/projects/index.js:565]: remove space at end

File: app/assets/stylesheets/projects.scss [hunk 2]
```diff
@@ -661,321 +661,35 @@ li.module-hover {
...
-    grid-row-gap: 1em;
-    grid-template-columns: repeat(auto-fit, minmax(270px, 1fr));
-    width: 100%;
-
-    .table-header {
-      display: none;
+.projects-index {
+  &.active {
+    [data-view-mode="archived"] {
+      display: none !important;
```
Comment (aignatov-bio) [inline app/assets/stylesheets/projects.scss:668]: Did you try without important?

File: app/assets/stylesheets/projects.scss [hunk 3]
```diff
@@ -661,321 +661,35 @@ li.module-hover {
...
     }
+  }
 
-    .projects-group {
-      grid-column: 1/-1;
-      margin: 0;
+  &.archived {
+    [data-view-mode="active"] {
+      display: none !important;
```
Comment (aignatov-bio) [inline app/assets/stylesheets/projects.scss:674]: Did you try without important?

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/projects/index.js (hunk 1 above)
Comment (urbanrotnik) [reply app/assets/javascripts/projects/index.js:565]: Woow 😱 Probably did this typo just before comitting or IDK how it was working...

File: app/assets/stylesheets/projects.scss (hunk 3 above)
Comment (urbanrotnik) [reply app/assets/stylesheets/projects.scss:674]: I did, not working.

File: app/assets/stylesheets/projects.scss (hunk 2 above)
Comment (urbanrotnik) [reply app/assets/stylesheets/projects.scss:668]: I did, not working.

File: app/assets/javascripts/projects/index.js [hunk 4]
```diff
@@ -563,28 +561,32 @@
...
-    });
-  }
-
   function initProjectsViewModeSwitch() {
-    $('input[type=radio][name=projects-view-mode-selector]').change(function() {
-      if (this.value === 'cards') {
+    let projectsPageSelector = '.projects-index';
+
+    // list/cards switch
+    $(projectsPageSelector + ' .projects-view-mode').click(function() {
```
Comment (aignatov-bio) [inline app/assets/javascripts/projects/index.js:568]: Use deligated event

File: app/assets/javascripts/projects/index.js [hunk 5]
```diff
@@ -563,28 +561,32 @@
...
         $('#cards-wrapper').removeClass('list');
-      } else if (this.value === 'list') {
+      } else if ($btn.hasClass('view-switch-list')) {
         $('#cards-wrapper').addClass('list');
       }
+      $btn.addClass('active');
+    });
+
+    // active/archived switch
+    $(projectsPageSelector + ' .archive-switch').click(function() {
```
Comment (aignatov-bio) [inline app/assets/javascripts/projects/index.js:580]: Use deligated event

File: app/assets/javascripts/projects/index.js [hunk 6]
```diff
@@ -682,7 +686,8 @@
       }
 
       $('.projects-filters').dropdown('toggle');
-      refreshCurrentView();
+
+      loadCardsView();
```
Comment (aignatov-bio) [inline app/assets/javascripts/projects/index.js:690]: refreshCurrentView()

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

**Review by aignatov-bio — APPROVED** [review]

**Pull request merged**